src/compiler/parser.ts

```diff
@@ -6468,7 +6468,7 @@ namespace ts {
 
                 function parseTagComments(indent: number) {
                     const comments: string[] = [];
-                    let state = JSDocState.SawAsterisk;

```

Comment:
Is this just a drive-by bug fix? I didn't see any real difference in parsing functionality.